rllib/algorithms/sac/sac.py

```diff
@@ -355,6 +355,7 @@ def validate(self) -> None:
             "EpisodeReplayBuffer",
             "PrioritizedEpisodeReplayBuffer",
             "MultiAgentEpisodeReplayBuffer",
+            "MultiAgentPrioritizedEpisodeReplayBuffer",
```

Comment:
Yeah, this is getting very long :D

Reply:
In the future we can remove all `Episode` in the names.